lisa/tools/lsvmbus.py

```diff
@@ -128,7 +128,9 @@ def install(self) -> bool:
             package_name = "linux-cloud-tools-common"
         if package_name:
             linux_os.install_packages(package_name)
-        else:
+        # 1 not all distro has above packages
+        # 2 some image has repo issue, then can't install package successfully
+        if not self._check_exists():
```

Comment:
Please also improve error message in executable.

`raise LisaException(f"install '{tool.name}' failed")`

it should be like 
`raise LisaException(f"install '{tool.name}' failed. After installed, it cannot be detected.")`